### ISSUE TITLE
Unwanted time played

### ISSUE THREAD

**Issue by brianx87** [benotter/TurnSignal#27]
I know it may not seem like a real issue but users like myself don't like to see small pieces of software clocking up hundreds of hours of playtime especially when it can interfere with the playtime of what's actually being played!. Can you please fix it so that TurnSignal runs in the background without it accumulating any playtime?

**Comment by SierraJC** [comment]
I agree with this, I am irritated by my steam ALWAYS saying I'm "In-App: Turn Signal" when I'm not even in VR anymore, and it annoys my friends when it pops up with it any time I exit a game.
At least release a binary on git so we can avoid the steam version.

**Comment by benotter** [comment]
Alright, I'll look more into it.

**Comment by benotter** [comment]
Heya, I'm actually getting around to fixing/adding this to the newest version, just wanted to give you guys a heads up on it.

**Comment by ApeFe** [comment]
Bump please. Love the software, not the annoyance that my friends have when they see me play. I now have all my notifications off so as to not bother my friends. Downside is that my friends don't know I'm playing games.

**Comment by benotter** [comment]
@ApeFe

Heya, thanks for the feedback!

So, it’s been kinda tough trying to get Steam to shut up about TurnSignal, like notifications and in-game time tracking,

That said, there is a beta branch available on Steam that disables steamworks using a similar method to other apps that can escape steams analytics and notifications, and includes several other small improvements!

You can access it by right clicking TurnSignal in the Steam Apps list, selecting properties, going to the beta tab, and selecting the beta from there,

If you give it a try and It doesn’t solve your issues, please feel free to hit me up here again, and i’ll take a look and see if I can further remove Steam’s annoyances!

Again, thanks for the feedback, feel free to let me know if you have any other Questions / Comments / Issues!

**Comment by terrehbyte** [comment]
I just switched to the beta branch and pulled the update before launching SteamVR again and it seems to still be logging time. The problem appears to still be that Steam considers me "playing" TurnSignal, which in turn contributes to the total amount of time played count stat that Steam displays on someone's profile.

![image](https://user-images.githubusercontent.com/4968773/52894359-f9d8d480-315c-11e9-8090-591a1a78ef0e.png)

I'm not familiar with SteamVR development enough to advise further, but would it be possible to get TurnSignal to launch without it registering as the current game I'm playing? Other VR utilities such as [OpenVR-AdvancedSettings](https://github.com/ykeara/OpenVR-AdvancedSettings) seem to be able to register as an application in SteamVR but doesn't count as the "currently played game" for the user.

In the meantime, as @NickSC mentioned before, making a standalone build accessible could be a good stopgap solution for those willing to manually download and run it when using SteamVR. Would that be possible?

**Comment by benotter** [comment]
@terrehbyte Hmm, if you haven't already, try disabling the Steamworks support option from the TurnSignal settings menu, on the next launch it should relaunch itself to escape Steam, and it should be able to be launched from its executable without checking with for the Steamworks API,

If this setting isn't working for you, I'd be interested in any details you can give me, as I've been using the disabled Steamworks option for a few months now under the beta, and its worked fairly well as far as I can see, but it still could have a few bugs or issues,

Again, please try disabling the option in the menu if you haven't, and let me know!

**Comment by terrehbyte** [comment]
Ah, I totally glossed over it in the main window. Sorry about that. Yeah, unchecking "Enable Steamworks" and then relaunching seems to work pretty much fine.

This is a non-issue for me, but Steam does briefly detect me in TurnSignal on initial launch before relaunching itself again without Steamworks. Closing SteamVR also closes TurnSignal whether Steamworks is enabled or not, which is perfect.

Thanks again for putting this together!
